design-system/materials/internals/definition.yaml

```diff
@@ -65,6 +65,7 @@ choiceGroupsByTheme:
         color-solid-05: 'hsl(0deg 0% 10% / 5%)'
         color-solid-10: 'hsl(0deg 0% 10% / 10%)'
         color-surface: '#fff'
+        color-transparent: 'transparent'
```

Comment:
Not sure that we need this color. I think we should use `color-surface` instead. That would help if we implement a dark theme in the future.

Reply:
My initial idea was to use `color-surface` but it introduced a regression. See the recording below. 
https://user-images.githubusercontent.com/6726467/207265047-53512bf1-7dd2-4151-befe-7f79f343a363.mov

Reply:
Yeah cause the border is defined in the parent container, not on the input.

I wouldn't add a new choice for that though, we can allow the values like `unset` or `transparent` to be used in the decision groups

Reply:
I updated to use `unset` instead - [302da5f](https://github.com/commercetools/ui-kit/pull/2347/commits/302da5ff5cfa26b531a68cd76fbfa6af3769387e)